submission-scripts/script-templates/magpie-magpie-customizations-substitution-tensorflow

```diff
@@ -0,0 +1,115 @@
+############################################################################
```

Comment:
I think you accidentally git added this file?  The "substitution" files shouldn't be added.